server/app/incident_service.go

```diff
@@ -778,9 +779,54 @@ func (s *IncidentServiceImpl) UpdateStatus(incidentID, userID string, options St
 		}()
 	}
 
+	if options.Status == StatusArchived && incidentToModify.ExportChannelOnArchiveEnabled {
+
+		fileID, err := s.exportFileToChannel(incidentToModify.Name, incidentToModify.OwnerUserID, incidentToModify.ChannelID)
+		if err != nil {
+			_, _ = s.poster.PostMessage(incidentToModify.ChannelID, "Incident Collaboration failed to export channel information. Contact your System Admin for more information.")
```

Comment:
I think we can remove the `information` word in this message.
```suggestion
			_, _ = s.poster.PostMessage(incidentToModify.ChannelID, "Incident Collaboration failed to export channel. Contact your System Admin for more information.")
```

Reply:
Done

Reply:
:tada: